### PR TITLE
Add meharshahani.md

### PR DESCRIPTION
**Pull request by meharshahani**
https://raw.githack.com/meharshahani/meharshahani.github.io/add-meharshahani-profile/#!pages/vi/profiles/meharshahani.md

### CODE REVIEW

File: pages/vi/profiles/meharshahani.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+# Hey all
+
+<ul>
```
Comment (lmmrssa) [inline pages/vi/profiles/meharshahani.md:4]: Use markdown instead of html

File: pages/vi/profiles/meharshahani.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <li>Mehar Shahani</li>  
+  <li>Chicago, IL, USA</li>
+  <li>Central Time Zone</li>
+  <li>MacOS Catalina</li>
+</ul>
+
+
+<img src = "https://media.timeout.com/images/105383118/1024/576/image.jpg " alt = "Chicago">
```
Comment (robinGiri) [inline pages/vi/profiles/meharshahani.md:12]: You can use markdown syntax for image too.

File: pages/vi/profiles/meharshahani.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  My name is **Mehar Shahani**. I am a graduate student in *Computer Science* at [Illinois Institute of Technology!](https://www.iit.edu/). I want to pursue a career in the field of Software/Android development. Looking forward to connecting with the team and contributing as quickly as possible.
+  
+  
+  ## Skills
+```
+Java, Android, Python, JavaScript, SQL, PHP, HTML, CSS, MongoDB 
+```
+  
+[LinkedIn profile](https://www.linkedin.com/in/meharshahani/) </br>
```
Comment (singharpita) [inline pages/vi/profiles/meharshahani.md:22]: Please remove html

Comment (meharshahani) [reply pages/vi/profiles/meharshahani.md]: @singharpita Done, please review it!!

File: pages/vi/profiles/meharshahani.md (hunk 1 above)
Comment (meharshahani) [reply pages/vi/profiles/meharshahani.md]: @lmmrssa Done!! Please review it.

**Pull request merged**